problem_builder/swipe.py

```diff
@@ -166,6 +140,20 @@ def expand_static_url(self, url):
                 pass
         return url
 
-    @property
-    def expanded_img_url(self):
-        return self.expand_static_url(self.img_url)
+    def mentoring_view(self, context=None):
+        """ Render the swipe image, text & whether it's correct within a mentoring block question. """
+        return Fragment(
+            (
+                u'<img src="{img_url}" style="max-width: 100%;" />'
+                u'<p class="swipe-text">"{text}"</p>'
+                u'<p>{answer}</p>'
```

Comment:
We shouldn't show the answer in the `mentoring_view` since that's the view that students would see (in theory). You could display `student_choice` here if you wanted (either "no choice made", "Selected True", or "Selected False"), or nothing (just text and image is fine).

Reply:
Ah right! I'll just remove the answer to keep it simple, then.